src/gap.c

```diff
@@ -876,6 +876,59 @@ Obj FuncRUNTIMES( Obj     self)
 }
 
 
+
+/****************************************************************************
+**
+*F  FuncMonotonicNanoseconds( <self> )
+**
+**  'FuncMonotonicNanoseconds' returns an integer which represents the 
+**  number of nanoseconds since some unspecified starting point. This means
+**  that the number returned by this function is not in itself meaningful,
+**  but the difference between the values returned by two consecutive calls
+**  can be used to measure wallclock time.
+**
+**  The accuracy of this is system dependent. For systems that implement
+**  clock_getres, we could get the promised accuracy.
+**
+**  Note that gettimeofday has been marked obsolete in the POSIX standard.
+**  We are using it because it is implemented in most systems still.
+**  Also, if we are using gettimeofday we cannot guarantee the values that
+**  are returned by MonotonicNanoseconds to be monotonic.
```

Comment:
So why call it `MonotonicNanoseconds` and not `Nanoseconds`? You don't explain what "monotonic" means in the documentation anyway, and this way, we don't raise expectations that we cannot satisfy on many systems.